test-harness/specs/E2E-Accept-Content-Negotiation-AC-15.oas2.txt

```diff
@@ -0,0 +1,24 @@
+====test====
+Sending multiple values in Accept header selects the first one matched.
+====spec====
+swagger: "2.0"
+paths:
+  /check-email:
+    get:
+      produces:
+        - application/json
+        - text/plain
+      responses:
+        "204":
+          description: OK, the email is available
+          examples:
+            application/json:
+              schema:
+                nullable: true
```

Comment:
This test might just show that no body is output when schema is a null, maybe the more standard `type: string` would create more confidence that this test is doing what we want for the reasons we want.

Reply:
I agree with @philsturgeon here and more importantly `204` responses do not have content and so Fastify will *delete it* no matter if you're trying to send it to the client, so let's change this to a regular 200.